src/open_inwoner/openklant/models.py

```
@@ -309,25 +323,24 @@ def __str__(self):
 
     @property
     def has_api_configuration(self) -> bool:
-        if self.primary_backend == KlantenServiceType.ESUITE.value:
-            esuite_config = ESuiteKlantConfig.get_solo()
-            return esuite_config.has_api_configuration()
-
-        # TODO: support `has_api_configuration` check for OK2?
-        return False
+        match self.primary_backend:
+            case KlantenServiceType.ESUITE.value:
+                config = ESuiteKlantConfig.get_solo()
+            case KlantenServiceType.OPENKLANT2.value:
+                config = OpenKlant2Config.get_solo()
+            case _:
+                config = None
```

I still feel this should be a `raise`: we don't want to silently allow unexpected values in the enum without this code being refactored.

---

Ah but of course this could not be set... my bad, ignore.